Density/app/src/main/java/org/cornelldti/density/density/facilitydetail/FacilityInfoPage.kt

```diff
@@ -92,9 +83,6 @@ class FacilityInfoPage : BaseActivity() {
         topBar.title = facilityClass!!.name
         topBar.setNavigationOnClickListener { onBackPressed() }
 
-        feedback = findViewById(R.id.accuracy)
-        feedback.movementMethod = LinkMovementMethod.getInstance()
```

Comment:
Is there a reason to remove accuracy rn? Aren't we planning to implement the feedback feature within a few weeks?

Reply:
yeaah but the current feedback system is kinda useless. Like we're gonna change it up entirely, so I figured we'd add it back once we had a new system. But keeping the old google form link just felt like it would be taking up space in the app.

Reply:
Okay gotcha!